### PR TITLE
RleImages

### PR DESCRIPTION
**Pull request by AndreyDeg**
Decode4BitRleImage and Decode8BitRleImage
Minor fix

### CODE REVIEW

**Comment by AndreyDeg** [comment]
I fix g/pal4rle.bmp, g/pal8rle.bmp and g/pal8topdown.bmp from http://entropymine.com/jason/bmpsuite/bmpsuite/html/bmpsuite.html

File: BMPDecoder.cs [hunk 1]
```diff
@@ -191,7 +207,7 @@ private void DecodeRGBQUAD(BinaryReader fileReader)
             quad.rgbB = Convert.ToInt16(0x00 | rgbB);
             quad.rgbG = Convert.ToInt16(0x00 | rgbG);
             quad.rgbR = Convert.ToInt16(0x00 | rgbR);
-            quad.rgbReserved = Convert.ToInt32(Convert.ToSByte(rgbReserved));
+            quad.rgbReserved = Convert.ToInt16(0x00 | rgbReserved);
```
Comment (AndreyDeg) [inline BMPDecoder.cs:210]: I have image with rgbReserved==255, and Convert.ToSByte failed for me

**Review by AndreyDeg — COMMENTED** [review]
Look it https://github.com/AndreyDeg/BMPDecoder/blob/Image/BmpDecoder.png?raw=true

File: BMPDecoder.cs (hunk 1 above)
Comment (EricHaung) [reply BMPDecoder.cs:210]: Thank you, bro.
You help me a lot.

**Pull request merged**